### ISSUE TITLE
Error while running the example python3 error_reporting_lalr.py

### ISSUE THREAD

**Issue by evandrocoan** [lark-parser/lark#228]
```
$ python3 error_reporting_lalr.py
Traceback (most recent call last):
  File "error_reporting_lalr.py", line 7, in <module>
    from .json_parser import json_grammar   # Using the grammar from the json_parser example
ModuleNotFoundError: No module named '__main__.json_parser'; '__main__' is not a package
```

I removed the dot on `from .json_parser` and now I got this:
```
$ python3 error_reporting_lalr.py
Traceback (most recent call last):
  File "d:\lark-parser\lark\parsers\lalr_parser.py", line 46, in get_action
    return states[state][key]
KeyError: '$END'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "error_reporting_lalr.py", line 34, in parse
    j = json_parser.parse(json_text)
  File "d:\lark-parser\lark\lark.py", line 223, in parse
    return self.parser.parse(text)
  File "d:\lark-parser\lark\parser_frontends.py", line 38, in parse
    return self.parser.parse(token_stream, *[sps] if sps is not NotImplemented else [])
  File "d:\lark-parser\lark\parsers\lalr_parser.py", line 83, in parse
    _action, arg = get_action('$END')
  File "d:\lark-parser\lark\parsers\lalr_parser.py", line 49, in get_action
    raise UnexpectedToken(token, expected, state=state)  # TODO filter out rules from expected
lark.exceptions.UnexpectedToken: Unexpected token '<EOF>' at line ?, column ?.
Expected one of:
        * RBRACE
        * COMMA


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "error_reporting_lalr.py", line 74, in <module>
    test()
  File "error_reporting_lalr.py", line 63, in test
    parse('{"example1": "value"')
  File "error_reporting_lalr.py", line 58, in parse
    raise exc_class(u.get_context(json_text), u.line, u.column)
  File "d:\lark-parser\lark\exceptions.py", line 21, in get_context
    start = max(pos - span, 0)
TypeError: unsupported operand type(s) for -: 'NoneType' and 'int'
```

I am running this against the latest version on your master branch.

Now if I revert back to the latest tag 0.6.4 and remove the . (dot) on `from .json`, it runs correctly:
```
$ python3 error_reporting_lalr.py
Missing Closing at line 1, column 14.

{"example1": "value"
             ^

Missing Opening at line 1, column 14.

{"example2": ]
             ^
```

**Comment by erezsh** [comment]
Hi, thanks for the bug report. This happened due to a recent fix (Issue #220)

I pushed a new commit to `master` that should fix this.

Btw, the way to run the examples is:

    python -m examples.error_reporting_lalr

**Comment by evandrocoan** [comment]
Now I am using this `python -m examples.error_reporting_lalr` and it is working, Thanks!
